v3iofs/fs.py

```diff
@@ -106,6 +162,64 @@ def touch(self, path, truncate=True, **kwargs):
             raise_for_status=[HTTPStatus.OK],
         )
 
+    def info(self, path, **kwargs):
+        """Give details of entry at path
+        Returns a single dictionary, with exactly the same information as ``ls``
+        would with ``detail=True``.
+        The default implementation should calls ls and could be overridden by a
+        shortcut. kwargs are passed on to ```ls()``.
+        Some file systems might not be able to measure the file's size, in
+        which case, the returned dict will include ``'size': None``.
+        Returns
```

Comment:
You need an empty line before `Returns`, see [here](https://numpydoc.readthedocs.io/en/latest/format.html)